app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+
+  # Return 3 most recent users
+  def recent_users
+    User.order(created_at: :desc).limit(3)
+  end
```

Comment:
- [ ] `User` model has method that returns **3 most recent posts**. You are supposed to retrieve the user's post but not the users. Let's fix that

Reply:
Thank you for your feedback. I will implement the changes.